app/Core/Db.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Core;
+
+class Db extends \PDO {
```

Comment:
It should be Singleton pattern, so some magic methods are supposed to be private, for example __construct